contracts/protocore/Protocore.sol

```diff
@@ -64,14 +64,58 @@ contract Protocore is MosaicVersion, ValidatorSet, CoconsensusModule {
     }
 
 
+    /* Constants */
+
+    /** EIP-712 domain separator name for Protocore. */
+    string public constant DOMAIN_SEPARATOR_NAME = "Mosaic-Core";
+
+    /** EIP-712 domain separator typehash for Protocore. */
+    bytes32 public constant DOMAIN_SEPARATOR_TYPEHASH = keccak256(
+        "EIP712Domain(string name,string version,bytes32 metachainId,address core)"
```

Comment:
Please make it consistent w.r.t Core contract's  `DOMAIN_SEPARATOR_TYPEHASH `

Reply:
Done